presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetReader.java

```diff
@@ -318,4 +324,31 @@ else if (ARRAY.equals(type.getTypeSignature().getBase())) {
         path.remove(name);
         return block;
     }
+
+    /**
+     * Helper used to determine the appropriate group annotation processing markers to use based on the schema of the parquet file being read.
+     *
+     * @param path The path to the parent column. This should be a map column.
+     * @return The value to use for the map group annotation in the column path.
+     */
+    @VisibleForTesting
```

Comment:
This is visible for testing, but no tests were added. Did you forget to include tests? If not, it would be good to write some tests for this functionality.

Reply:
Please add some tests for this functionality. You can start by looking at `AbstractTestHiveFileFormats` and `ParquetTester` (it has a quick (`quickParquetTester()`) and a full (`fullParquetTester()`) tester).